src/main/java/io/tus/java/client/TusClient.java

```diff
@@ -115,6 +116,39 @@ public boolean removeFingerprintOnSuccessEnabled() {
         return removeFingerprintOnSuccessEnabled;
     }
 
+    /**
+     * Enable chunked transfer encoding.
```

Comment:
Could you expand the comment a bit to explain what the advantages/disadvantages/impacts of enabling/disabling chunked transfer encoding has?